src/components/task-plan/reading.cjsx

```diff
@@ -82,91 +99,99 @@ SelectTopics = React.createClass
       </div>
     </BS.Modal>
 
+ReadingFooter = React.createClass
+  render: ->
+    classes = []
+    classes.push('disabled') unless @props.enabled
```

Comment:
I'll add in the propTypes, but we should probably have a sprint task/pull request somewhere to clean those up too.